Create 2019-12-26-countdown-to-the-new-year-ember-cli-update.md (#452)
ember-learn/ember-blog: source/2019-12-26-countdown-to-the-new-year-ember-cli-update.md

---
title: Countdown to The New Year - Ember CLI Update
author: Kelly Selden
tags: 2019, Addons, DecEmber
alias: 2019-12-26-countdown-to-the-new-year-ember-cli-update
responsive: true
---

**This is the 26th in our December series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**

## Day 26

> Let's highlight how easy it is to update Ember projects - [ember-cli-update](https://emberobserver.com/addons/ember-cli-update).

Reviewer: ```suggestion
Let's highlight how you can update Ember projects swiftly - with [ember-cli-update](https://emberobserver.com/addons/ember-cli-update).
```

READMORE

### What It Does

Besides package updates like ember-source and ember-cli, there are more files that are updated when a new version of Ember releases.
Tweaks to testem.js or .eslintrc.js may be missed if you're not tracking those files changes (via something like [ember-new-output](https://github.com/ember-cli/ember-new-output)).
ember-cli-update was created to update those kinds of files along with package.json changes for you.
It can also run codemods against your newly installed versions.

> A new feature is on its way where it can maintain any blueprint, not just the official Ember blueprint.

Reviewer: ```suggestion
A new feature is on its way where it can maintain not only the official Ember blueprint, but any custom blueprint.
```

> The idea is that you can make one blueprint for your organization, and keep your projects up-to-date just like ember-cli-update does today.

Reviewer: ```suggestion
The idea is that you can make one blueprint for your organization, and keep your projects up-to-date in the same manner as ember-cli-update does today.
```
